source/recreation.rst

```diff
@@ -7,7 +7,7 @@ Visitation: Recreation and Tourism
 Summary
 =======
 
-Recreation and tourism are important components of many national and local economies and they contribute in innumerable ways to quality of life, sense of place, social connection, physical wellbeing, learning, and other intangibles.  To quantify the value of natural environments, the InVEST recreation model predicts the spread of person-days of recreation, based on the locations of natural habitats and other features that factor into people's decisions about where to recreate.  The tool estimates the contribution of each attribute to visitation rate in a simple linear regression.  In the absence of empirical data on visitation, we parameterize the model using a proxy for visitation: geotagged photographs posted to the website flickr.  Using photo-user-day estimates, the model predicts how future changes to natural features will alter visitation rates.  The tool outputs maps showing current patterns of recreational use and maps of future patterns of use under alternate scenarios.
+Recreation and tourism are important components of many national and local economies and they contribute in innumerable ways to quality of life, sense of place, social connection, physical wellbeing, learning, and other intangibles. To quantify the value of natural environments, the InVEST recreation model predicts the spread of person-days of recreation, based on the locations of natural habitats and other features that factor into people's decisions about where to recreate. The tool estimates the contribution of each attribute to visitation rate in a simple linear regression. In the absence of empirical data on visitation, we parameterize the model using a proxy for visitation: geotagged photographs posted to the website flickr. Using photo-user-day estimates, the model predicts how future changes to natural features will alter visitation rates. The tool outputs maps showing current patterns of recreational use and maps of future patterns of use under alternate scenarios.
```

Comment:
Just checking that the move to single space vs double space for sentences was a choice and not an editor preference change.

Reply:
Yeah, I did it on purpose. A lot of files had some double spaces mixed in, but mostly used single space. Super minor but it gets annoying once you start noticing it!